src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/notification/CaseHandedOfflineDefendantNotificationHandler.java

```diff
@@ -0,0 +1,73 @@
+import uk.gov.hmcts.reform.unspec.callback.Callback;
+import uk.gov.hmcts.reform.unspec.callback.CallbackHandler;
+import uk.gov.hmcts.reform.unspec.callback.CallbackParams;
+import uk.gov.hmcts.reform.unspec.callback.CallbackType;
+import uk.gov.hmcts.reform.unspec.callback.CaseEvent;
+import uk.gov.hmcts.reform.unspec.config.properties.notification.NotificationsProperties;
+import uk.gov.hmcts.reform.unspec.helpers.CaseDetailsConverter;
+import uk.gov.hmcts.reform.unspec.model.CaseData;
+import uk.gov.hmcts.reform.unspec.service.NotificationService;
+
+import java.util.List;
+import java.util.Map;
+
+import static uk.gov.hmcts.reform.unspec.callback.CaseEvent.NOTIFY_RESPONDENT_SOLICITOR1_FOR_CASE_HANDED_OFFLINE;
+import static uk.gov.hmcts.reform.unspec.handler.callback.notification.NotificationData.CLAIM_REFERENCE_NUMBER;
+import static uk.gov.hmcts.reform.unspec.handler.callback.notification.NotificationData.SOLICITOR_NAME;
+
+@Service
+@RequiredArgsConstructor
+public class CaseHandedOfflineDefendantNotificationHandler extends CallbackHandler {
+
+    private static final List<CaseEvent> EVENTS = List.of(NOTIFY_RESPONDENT_SOLICITOR1_FOR_CASE_HANDED_OFFLINE);
+    public static final String NOTIFY_RESPONDENT_SOLICITOR1_FOR_CASE_HANDED_OFFLINE_TASK_ID =
+        "NotifyDefendantSolicitorForCaseHandedOffline";
+    private static final String REFERENCE_TEMPLATE = "case-handed-offline-defendant-notification-%s";
+
+    private final NotificationService notificationService;
+    private final NotificationsProperties notificationsProperties;
+    private final CaseDetailsConverter caseDetailsConverter;
+
+    @Override
+    protected Map<CallbackType, Callback> callbacks() {
+        return Map.of(
+            CallbackType.ABOUT_TO_SUBMIT, this::notifyDefendantSolicitorForCaseHandedOffline
+        );
+    }
+
+    @Override
+    public String camundaActivityId() {
+        return NOTIFY_RESPONDENT_SOLICITOR1_FOR_CASE_HANDED_OFFLINE_TASK_ID;
+    }
+
+    @Override
+    public List<CaseEvent> handledEvents() {
+        return EVENTS;
+    }
+
+    private CallbackResponse notifyDefendantSolicitorForCaseHandedOffline(CallbackParams callbackParams) {
+        CaseData caseData = caseDetailsConverter.toCaseData(callbackParams.getRequest().getCaseDetails());
+
+        notificationService.sendMail(
+            "defendant-solicitor@example.com",
+            notificationsProperties.getSolicitorResponseToCase(),
+            getNotificationProperties(caseData),
+            String.format(REFERENCE_TEMPLATE, caseData.getLegacyCaseReference())
+        );
+
+        return AboutToStartOrSubmitCallbackResponse.builder().build();
+    }
+
+    private Map<String, String> getNotificationProperties(CaseData caseData) {
+        return Map.of(
+            CLAIM_REFERENCE_NUMBER, caseData.getLegacyCaseReference(),
+            SOLICITOR_NAME, caseData.getSolicitorReferences().getRespondentSolicitor1Reference()
```

Comment:
`SOLICITOR_NAME` or `SOLICITOR_REFERENCE` ?

Reply:
should be reference, updated in all places